.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,35 @@
+name: Unit Test
+
+on:
+  push:
+    branches:
+    - 2.0.x
+  pull_request:
+    branches:
+    - 2.0.x
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [10, 12]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+
+    - run: |
+        npm install
+        npm run lint
+        npm run test
+      working-directory: ask-sdk-core
```

Comment:
For "ask-sdk",
    "ask-sdk-core",
    "ask-sdk-dynamodb-persistence-adapter",
    "ask-sdk-runtime",
    "ask-sdk-s3-persistence-adapter",
    "ask-sdk-v1adapter",
could run 
 npm install
 npm run bootstrap
 npm run gulp
under the top level directory. 
These six packages are under the big lerna project, and above command will install dependencies and run unit tests one by one in each pacakge.

Reply:
Yes, The plan is to migrate all packages on using workflows alone and not depends on gulp, lerna.
But I will research if i can trigger all packages lints and tests once I've made changes across :)